conf/prod.routes

```diff
@@ -1,6 +1,6 @@
 # Add all the application routes to the app.routes file
-->         /eeitt-admin-frontend              app.Routes
-->         /                          health.Routes
-->     	   /template                  template.Routes
-
-GET        /admin/metrics             com.kenshoo.play.metrics.MetricsController.metrics
+->         /submissions                  app.Routes
```

Comment:
`/submission` is dedicated to gforms, it should not be used.

Reply:
sure, will update